vagga.yaml

```diff
@@ -0,0 +1,115 @@
+containers:
+  mysql:
+    setup:
+      - !Alpine v3.4
+      - !Install
+        - mariadb
+        - mariadb-client
+      - !EnsureDir /data
+    environ: 
```

Comment:
Because
a) Flexibility
b) environ file looks like temporary workaround until config is simplified to a single file. I do not want to spend time on adding extra logic to its creation